scripts/third_party_licenses/apache.tpl

```diff
@@ -0,0 +1,5 @@
+{{range . -}}
+{{if eq .LicenseName "Apache-2.0" -}}
+** {{.Name}}; version {{.Version}} - https://{{.Name}}
```

Comment:
Is every name really likely to be a valid `https://` domain?

Reply:
Good catch. The few I checked resolved to the Go packages page, but not every package would resolve. Instead this should be `{{.LicenseURL}}`

Reply:
Pushed with latest. PTAL.